doc/rados/operations/health-checks.rst

```diff
@@ -301,6 +301,30 @@ You can either raise the pool quota with::
 
 or delete some existing data to reduce utilization.
 
+BLUEFS_SPILLOVER
+________________
+
+One or more OSDs that use the BlueStore backend have been allocated
+`db` partitions (storage space for metadata, normally on a faster
+device) but that space has filled, such that metadata has "spilled
+over" onto the normal slow device.  This isn't necessarily an error
+condition or even unexpected, but if the administrator's expectation
+was that all metadata would fit on the faster device, it indicates
+that not enough space was provided.
+
+This warning can be disabled with::
+
+  ceph config set osd bluestore_warn_on_bluefs_spillover false
```

Comment:
Shouldn't we mention that this can be disabled on per-OSD basis as well?

Reply:
updated!